pyproject.toml

```diff
@@ -4,7 +4,7 @@ name = "MyoSuite"
 dynamic = ["version"]  # Enable dynamic version
 description = "Musculoskeletal environments simulated in MuJoCo"
 readme = "README.md"
-requires-python = ">=3.9"
+requires-python = ">=3.10"
```

Comment:
do we need python 3.10? all the latest version of mujoco-mjx should be compatible with python 3.9 too

Reply:
Indeed, the mujoco-mjx is compatible with 3.9+; the stricter version requirement here comes from [mujoco_playground](https://github.com/google-deepmind/mujoco_playground/blob/60cf8430011da531e08ce95a6ea48a56e5fbad7c/pyproject.toml#L12), which needs Python 3.10 or newer

Reply:
@Balint-H do you know if it is possible to keep 3.9 if we use just the wrappers?